### PR TITLE
[stable/phabricator] Add nodeSelector, affinity and tolerations

### PR DESCRIPTION
**Pull request by uzxmx**
#### What this PR does / why we need it:

Add settings that control pod assignment.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by UzxMx** [comment]
@prydonius  @tompizmor  @juan131  Could you help to review this PR?

File: stable/phabricator/values.yaml [hunk 1]
```diff
@@ -237,3 +237,15 @@ metrics:
   ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
   ##
   # resources: {}
+
```
Comment (carrodher) [inline stable/phabricator/values.yaml:240]: Jus for comments convention, can you add a `##` line between the variable and the comment? And also use two # for comments:
```diff
+ ## Node labels for pod assignment
+ ## Ref: https://kubernetes.io/docs/user-guide/node-selection/
+ ##
  nodeSelector: {}
```

File: stable/phabricator/templates/deployment.yaml [hunk 2]
```diff
@@ -167,4 +167,16 @@ spec:
...
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```
Comment (carrodher) [inline stable/phabricator/templates/deployment.yaml:181]: In other charts, we are using a pretty similar format but not the same:
```yaml
    {{- if .Values.nodeSelector }}
      nodeSelector:
{{ toYaml .Values.nodeSelector | indent 8 }}
      {{- end -}}
    {{- with .Values.affinity }}
      affinity:
{{ toYaml . | indent 8 }}
    {{- end }}
    {{- with .Values.tolerations }}
      tolerations:
{{ toYaml . | indent 8 }}
```
Can we homogenize it?

**Comment by carrodher** [comment]
Thanks for the PR, I just added small comments, but in general it looks good to me

File: stable/phabricator/values.yaml (hunk 1 above)
Comment (uzxmx) [reply stable/phabricator/values.yaml:240]: Just updated.

File: stable/phabricator/templates/deployment.yaml (hunk 2 above)
Comment (uzxmx) [reply stable/phabricator/templates/deployment.yaml:181]: Just updated.

**Comment by UzxMx** [comment]
@carrodher  Just updated as you commented. Feel free to send any feedback to me.

**Pull request merged**